en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md

```diff
@@ -1,126 +1,129 @@
-    |artifact_retriever|  we can specify the extension point. The default is `DBRetriever` where the artifacts are pulled from the database.|
-    |deployment_retry_duartion|We can specify the retry duration in milliseconds to deploy artifacts, if there is a failure in pulling them from the  extension through `deployment_retry_duartion`. The retry duration specified here will be exponentially increased by a progression factor of 2. That means the duration will be progressed as 15s, 30s, 60s, 120s……. if there are continuous failures. And retry duration is bounded with 1 hr.<html><div class="admonition note"><p class="admonition-title">Note</p><p> This`deployment_retry_duartion` is  applicable only in Asynchronous deployment (`data_retrieval_mode ="async"` )where the server will try todeploy the artifacts after it is started. </p></div></html>|
-    |data_retrieval_mode|Through `data_retrieval_mode = “sync” ` we can specify the mode of deployment of artifacts from the extension point. By default gateway Startup will be in a Synchronous manner. Here the server will wait until all the APIs have been deployed. If there is any failure in the deployment, it will again be triggered for n times where n is the maximum retry count. If the APIs are not deployed even in n retries then the server will start with un deployed API artifacts. If the user wants to switch the startup mode to an Asynchronous mode then he needs to specify it in the configs as `data_retrieval_mode = "async"`. Here the server will be up, independent of the deployment of synapse artifacts in gateway.|
-     |save_artifacts_locally|If we add the config `save_artifacts_locally = true` or remove `save_artifacts_locally` config  then synapseartifacts will be stored in the file system. (Saved in `<APIM_HOME>/repository/deployment/server/synapse-configs/default/`) . When `save_artifacts_locally = false` then the artifacts from extension point will not be stored in the file system.|
+    |gateway_labels|Specify the labels that the Gateway is going to subscribe to. Only the APIs with these labels will be pulled from the extension point and deployed.|
+    |artifact_retriever|  Specify the extension point. The default is `DBRetriever` where the artifacts are pulled from the database.|
+    |deployment_retry_duartion| Specify the retry duration in milliseconds to deploy artifacts if there is a failure in pulling them from the  extension through `deployment_retry_duartion`. The retry duration specified here will be exponentially increased by a progression factor of 2. Thereby, the duration will be progressed as 15s, 30s, 60s, 120s and so on if there are continuous failures. The retry duration is bounded by 1 hr.<html><div class="admonition note"><p class="admonition-title">Note</p><p> This`deployment_retry_duartion` is  applicable only in Asynchronous deployment (`data_retrieval_mode ="async"`) where the server will try to deploy the artifacts after it is started. </p></div></html>|
```

Comment:
typo? deployment_retry_duartion

Reply:
No. It is a config element. For example

```
[apim.sync_runtime_artifacts.gateway]
     gateway_labels =["Production and Sandbox","Label1","Label2"]
     artifact_retriever = "DBRetriever"
     deployment_retry_duartion = 15000
     data_retrieval_mode = "sync"
     save_artifacts_locally = false
```

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1796